app/views/layouts/footer.html.erb

```diff
@@ -0,0 +1,65 @@
+<section class="return-to-top">
+  <div class="grid-container grid-container-desktop usa-footer__return-to-top">
+    <div class="grid-row">
+      <div class="grid-col-auto">
+        <a href="#"><span>Return to top</span> <i class="fas fa-arrow-circle-up"></i></a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<div class="usa-identifier">
+  <section
+    class="usa-identifier__section usa-identifier__section--masthead"
+    aria-label="Agency identifier">
+    <div class="usa-identifier__container"><div class="usa-identifier__logos">
+        <a href="javascript:void(0);" class="usa-identifier__logo">
+          <img
+            class="usa-identifier__logo-img"
+            
+            src="<%= asset_path('gsa-logo-w100.png') %>"
+            
+            
+            alt="&lt;GSA&gt; logo"
+            
+            role="img">
+        </a></div><div class="usa-identifier__identity" aria-label="Agency description">
+        <p class="usa-identifier__identity-domain">sorndash.fpc.gov</p>
```

Comment:
I'll leave this up to you, but maybe we ought to change it to our cloud.gov url until we have the new url

Reply:
sure